lumicks/pylake/kymo.py

```diff
@@ -586,18 +589,40 @@ def estimate_bead_edges(
             any prior deprecation notice! If you use this functionality keep a close eye on the
             changelog for any changes that may affect your analysis.
 
+        There are two algorithms to determine bead edges:
+
+            - "brightness": brightness based bead edge detection
+            - "template": template correlation-based bead edge detection
+
+        1. Brightness
+
+        Searches for bead edges by summing along the temporal direction and removing small features.
+        The result is smoothed and peaks are detected. Bead edges are found by checking where the
+        fluorescence drops below a threshold.
+
+        2. Template
+
+        Downsamples the kymograph to a specific number of frames (specified with the optional
+        parameter `num_time_frames`). For each frame, we traverse the scan line pixel by pixel,
+        extracting a template at each pixel. This template is then correlated with the next frame.
+        This process is repeated until all frames have been processed and a 2D matrix is acquired.
+        This matrix is then summed over the temporal axis obtaining a similarity score over time.
+        This score can be interpreted as a measure for how long a template taken at that particular
+        location is present. The outer maxima of this curve provide the bead centers.
```

Comment:
This looks odd to me when rendered. It's difficult to tell if the next paragraph applies to the "template" option still or to both
<img width="568" alt="image" src="https://github.com/lumicks/pylake/assets/61475504/0f10451c-3f35-4156-96a2-1bdf84d61594">

I would suggest something like this
<img width="538" alt="image" src="https://github.com/lumicks/pylake/assets/61475504/4728f6c1-5a37-4db1-ab8f-316eb049abac">

```suggestion
        * *Brightness: brightness based bead edge detection.*
          Searches for bead edges by summing along the temporal direction and removing small features.
          The result is smoothed and peaks are detected. Bead edges are found by checking where the
          fluorescence drops below a threshold.

        * *Template: template correlation-based bead edge detection.*
          Downsamples the kymograph to a specific number of frames (specified with the optional
          parameter `num_time_frames`). For each frame, we traverse the scan line pixel by pixel,
          extracting a template at each pixel. This template is then correlated with the next frame.
          This process is repeated until all frames have been processed and a 2D matrix is acquired.
          This matrix is then summed over the temporal axis obtaining a similarity score over time.
          This score can be interpreted as a measure for how long a template taken at that particular
          location is present. The outer maxima of this curve provide the bead centers.
```